### DISCUSSION TITLE
How to use 3D Model?

### DISCUSSION THREAD

**Discussion by cubevis** [maplibre/maplibre-react-native#498] [Q&A] [answered]
How to use 3D Model?

**Comment by KiwiKilian** [comment, marked as answer]
AFAIK there is not 3D support yet the MLRN, not sure what's possible with MLN. You can use extrusion with [`FillExtrusionLayer`](https://github.com/maplibre/maplibre-react-native/blob/main/docs/FillExtrusionLayer.md) which is used in this [example]( https://github.com/maplibre/maplibre-react-native/blob/main/packages/examples/src/examples/FillRasterLayer/IndoorBuilding.tsx).

**Reply by yerffejytnac** [reply]
Any plans to support custom layers like in this example? https://maplibre.org/maplibre-gl-js/docs/examples/add-a-3d-model-to-globe-using-threejs/
